torchbenchmark/models/BERT_pytorch/bert_pytorch/dataset/dataset.py

```diff
@@ -16,7 +16,8 @@ def __init__(self, corpus_path, vocab, seq_len, encoding="utf-8", corpus_lines=N
         self.corpus_path = corpus_path
         self.encoding = encoding
 
-        with open(corpus_path, "r", encoding=encoding) as f:
+        with generator as f:
```

Comment:
Does `tqdm.tqdm()` correctly handle the case where generator is `None`? If not, please add an assert here that `f` is not `None`.